### PR TITLE
Get rid of `app/resources`

### PR DESCRIPTION
**Pull request by tpendragon**
This is a much simpler version of #4978 which was actually completable.

This just separates out change_sets/decorators/wayfinders/controllers into their own directories again.

### CODE REVIEW

File: spec/change_sets/recording_change_set_spec.rb [hunk 1]
```diff
@@ -2,106 +2,31 @@
 require "rails_helper"
 
 RSpec.describe RecordingChangeSet do
-  subject(:change_set) { described_class.new(form_resource) }
```
Comment (hackartisan) [inline spec/change_sets/recording_change_set_spec.rb:5]: Why are there a bunch of tests removed? Are they duplicates or something? I wonder if this should be in a separate PR.

**Review by hackartisan — CHANGES_REQUESTED** [review]
One question but once the test suite passes I'm 👍🏻

File: spec/change_sets/recording_change_set_spec.rb (hunk 1 above)
Comment (tpendragon) [reply spec/change_sets/recording_change_set_spec.rb]: Good catch - we had two recording_change_set specs, so my renaming killed one of them. They're combined now.

**Review by eliotjordan — APPROVED** [review]
